.circleci/config.yml

```diff
@@ -414,6 +414,26 @@ jobs:
       - checkout
       - run: *run_docs_pragma_min_version
 
+  t_pyscripts_ubu:
+    docker:
+      - image: << pipeline.parameters.ubuntu-2004-docker-image >>
+    steps:
+      - checkout
+      - run:
+          name: Python unit tests
+          command: python3 test/pyscriptTests.py
+
+  t_pyscripts_win:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - run: git config --global core.autocrlf false
+      - checkout
+      - run:
+          name: Python unit tests
+          command: python.exe test/pyscriptTests.py
```

Comment:
Same here

Reply:
I haven't checked if `python3.exe` exists but `python.exe` does.

Reply:
Well, let's try and see. I changed that to `python3.exe` now.

Reply:
No `python3.exe` :)